usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -57,6 +58,7 @@
         "custom_schema": "recipient_hash STRING, federal_accounts STRING, cfdas ARRAY<STRING>,"
         " tas_components ARRAY<STRING>",
         "column_names": list(AWARD_SEARCH_COLUMNS),
+        "tsvectors": None,
```

Comment:
We're having a lot of dict keys pop up. A good unit test would be to fetch the `TABLE_SPEC` (combined), and throw an error if any key is in one entry that's missing from any other entry.

Reply:
Added a quick unit test for this.